Dont set MAKE=make, this breaks if we invoke it with say gmake and make is bsd
make.

https://bugzilla.redhat.com/show_bug.cgi?id=461597


--- configure.in.old	2008-06-01 16:24:13 +0000
+++ configure.in	2008-06-01 16:24:23 +0000
@@ -7,8 +7,6 @@
 AC_PROG_CC
 AC_ISC_POSIX
 AC_PROG_LN_S
-AC_PATH_PROG(MAKE,make)
-AC_PROG_MAKE_SET
 AC_PROG_AWK
 
 AM_PROG_LIBTOOL
